browser/themes/brave_theme_service_browsertest.cc

```diff
@@ -194,7 +194,12 @@ IN_PROC_BROWSER_TEST_F(BraveThemeServiceTest, OmniboxColorTest) {
 // And Toggle it twice from initial value to go back to initial value  because
 // reg value changes system value. Otherwise, dark mode config could be changed
 // after running this test.
+#if defined(ARCH_CPU_X86)
```

Comment:
FYI - we have a convention for this situation :) 
```
#if BUILDFLAG(IS_WIN) && defined(ARCH_CPU_X86)
#define MAYBE_DarkModeChangeByRegTest \
  DISABLED_DarkModeChangeByRegTest
#else
#define MAYBE_DarkModeChangeByRegTest \
  DarkModeChangeByRegTest
#endif
IN_PROC_BROWSER_TEST_F(BraveThemeServiceTest,
                       MAYBE_DarkModeChangeByRegTest) {
```

Reply:
thanks, @simonhong 

I wasn't aware and force-pushed fix